pkgs/os-specific/linux/nvidia-x11/default.nix

```diff
@@ -83,20 +83,11 @@ rec {
     broken = kernel.kernelAtLeast "5.18";
 
     patches =
-      let patch390 = o:
-        (lib.optional ((lib.versions.majorMinor kernel.modDirVersion) == o.version) (fetchpatch {
-          inherit (o) sha256;
-          url = "https://gitlab.com/herecura/packages/nvidia-390xx-dkms/-/raw/herecura/kernel-${o.version}.patch";
-        }));
-      in
-        []
-        ++ (patch390 {
+      (patch390 {
           version = "5.18";
           sha256 = "sha256-A6itoozgDWmXKQAU0D8bT2vUaZqh5G5Tg3d3E+CLOTs=";
-        })
-      ;
+      });
   };
```

Comment:
I would probably refactor the patch logic into a separate diff.

Odd to have a bump and significant refactor as part of the same commit. Makes conflicts more likely when backporting.

Reply:
Done, I'll pick 2bb6c5b24e0eb6ff3e778204de380d98e8395909 in a future PR.